lib/surveyor/models/survey_translation_methods.rb

```diff
@@ -7,7 +7,7 @@ module SurveyTranslationMethods
 
       included do
         # Associations
-        belongs_to :survey
+        belongs_to :survey, required: false
```

Comment:
how does a translation for a survey not require a survey?

Reply:
I'm just keeping parity with previous behavior.